src/config/firebaseAdmin.ts

```diff
@@ -0,0 +1,14 @@
+import { applicationDefault, initializeApp } from 'firebase-admin/app';
+import * as admin from 'firebase-admin';
+
+initializeApp({
+    credential: applicationDefault()
+});
+
+export const deleteEmployee = async (employeeFirebaseId: string) => {
+    try {
+        await admin.auth().deleteUser(employeeFirebaseId);
+    } catch (e) {
+        console.log(e);
```

Comment:
should do error handling on top of `console.log`
because when I provided a fake non-existent firebase_id, the backend will crash